index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```

Comment:
If this is just for demo purposes, can we not put it in the project root? Seems counterintuitive to be seeing stuff like this when you first see the project. Same goes for webpack config if it's just for demo purposes.